pkg/circular/circular_test.go

```diff
@@ -17,15 +17,15 @@ import (
 	"github.com/stretchr/testify/suite"
 	"golang.org/x/time/rate"
 
-	"github.com/talos-systems/talos/internal/pkg/circular"
+	circular2 "github.com/talos-systems/talos/pkg/circular"
```

Comment:
wth? :)

Reply:
Refactoring didn't work as good as I expected... One instance of it I fixed, but other slipped.